Add Kata Containers runtimes to libpod.conf

saschagrunert opened the pull request:
This adds the Kata Containers runtimes to the libpod.conf and adds
additional documentation to it.

Note: openSUSE Tumbleweed packages Kata where it works out of the box if the recommended packages got installed, too. If you want, then I could add instructions to the README how to setup podman together with Kata as a follow up, but I'm not sure which other distributions work as well. Maybe some of the Red Hat maintained ones?

rhatdan commented on libpod.conf:
```diff
@@ -146,6 +146,23 @@ crun = [
+kata-runtime = []
+
+# Kata Containers with the QEMU VMM
+kata-qemu = []
```
Shouldn't you have the default paths in here?

saschagrunert replied:
They will be discovered by the `$PATH` env variable, should I add them nevertheless?

rhatdan replied:
The other OCI runtimes paths are mentioned.  If Fedora decides to install kata in a non standard path, /usr/lib/libexec/kata/kata or something like that, then we would want these paths initialized for the distribution.  I am not sure all distros want to support the kata executable directly or rather as a library/executable for other packages like CRI-O, Podman ...

rhatdan commented:
@saschagrunert Any time to fix this up?

saschagrunert commented:
> @saschagrunert Any time to fix this up?

Yes sure, this has somehow dropped from my notification list. I rebased on top of the latest master and added the absolute paths for the runtimes.

mheon commented:
/approve

Change LGTM

rhatdan commented:
/lgtm
